pkg/graveler/graveler.go

```diff
@@ -1880,6 +1880,7 @@ func (g *Graveler) LoadCommits(ctx context.Context, repositoryID RepositoryID, m
 	if iter.Err() != nil {
 		return iter.Err()
 	}
+	// after removing this code, we won't be able to support dump that created using lakFS version that's lower than v0.61.0
```

Comment:
Neat comment!  May I suggest:
1. Open an issue to describe what you intend to do (later).  Maybe even write a suggested date for it, and/or set a reminder on your calendar to do it in 9 months or whatever you decide.
1. Reference that issue in your comment.  The way I like to do it is to write something like
   ```go
   // TODO(#27183): Remove this code to drop support for dumps created by lakeFS versions below v0.61.0.
   ```
   (assuming the issue you opened got number 27183).